buildpack.md

```diff
@@ -465,8 +474,9 @@ During the build phase, typical buildpacks might:
 1. Compile the application source code into object code.
 1. Remove application source code that is not necessary for launch.
 1. Provide start command in `<layers>/launch.toml`.
-1. Write a partial Bill-of-Material to `<layers>/launch.toml` describing any provided application dependencies.
-1. Write a partial Bill-of-Material to `<layers>/build.toml` describing any provided build dependencies.
+1. Write a partial Software Bill of Materials to `<layers>/<layer>.sbom.<ext>` describing any dependencies provided in the layer.
+1. Write a partial Software Bill of Materials to `<layers>/launch.sbom.<ext>` describing any provided application dependencies not associated with a layer.
+1. Write a partial Software Bill of Materials to `<layers>/build.sbom.<ext>` describing any provided build dependencies not associated with a layer.
```

Comment:
I think one thing that I know people have gotten confused about, is where they should write sbom for libraries that were included in an output compiled binary. For eg go libraries in a go binary. Those should technically be in the launch sbom as they are present in the final app image. On the other hand tools like a package manager might be better off in the build sbom. Not sure if we should clarify here or in the docs.